src/main/java/com/sforce/ws/codegen/Generator.java

```diff
@@ -70,27 +69,20 @@
     public final static String TYPE = "type";
     public final static String TYPE_INTERFACE = "typeinterface";
 
-    protected final TypeMapper typeMapper = new TypeMapper();
+    protected final TypeMapper typeMapper;
     protected final ArrayList<File> javaFiles = new ArrayList<File>();
     protected final String packagePrefix;
-    protected final String interfacePackagePrefix;
 
     protected final STGroupDir templates;
     protected boolean generateInterfaces;
 
-    public Generator(String packagePrefix, STGroupDir templates, String interfacePackagePrefix) throws Exception {
```

Comment:
Please do not delete public constructors, feel free to add new constructors.

Reply:
🆗

Reply:
@reubencornel I have restored two old constructor. However, it indeed looks very suspicious to see a constructor with two last parameters ignored (`char startDelim, char endDelim`). Is it really needed, used anywhere, essential for compatibility?

Reply:
Its mostly for compatibility.